### DISCUSSION TITLE
Release on elementary OS AppCenter

### DISCUSSION THREAD

**Discussion by moonmehta** [Ulauncher/Ulauncher#887] [Packaging]
There's a closed bug on this (#199), but the premise for closing the bug is not valid since AppCenter apps don't have to be coded in Vala. See excerpt below:

> We frequently get asked if developers can publish apps written in languages other than Vala or with a build system other than CMake. The answers is “yes”! All of our documentation is built around the “one true development path” that elementary follows, but as long as your app is a native Gtk3 app with Debian packaging, it should be publishable. Several apps published in AppCenter are already using the Meson build system, for example.

Source: https://medium.com/elementaryos/so-you-wanna-publish-on-appcenter-12c675b56e1c#aec2

**Comment by CameronNemo** [comment]
AppCenter also uses debian packaging, which already exists for ulauncher. Should not need many additions to submit to AppCenter.

**Comment by gornostal** [comment]
Looking for contributors who can figure out how to package for Elementary OS and provide either a proof of concept or a step-by-step guide.

**Comment by CameronNemo** [comment]
@gornostal if you have Debian packaging already it is a matter of integrating with their CI (Houston).

**Comment by gornostal** [comment]
@CameronNemo  thanks for the info

**Comment by CameronNemo** [comment]
Seems we are at the point of needing to write the appdata file, as described in [this section](https://github.com/elementary/houston/wiki/Before-You-Publish#appdata-and-oars) of the elementary publishing/packaging wiki.
